updatecli/updatecli.d/grafana.yaml

```
@@ -0,0 +1,53 @@
+sources:
+  lastRelease:
+    kind: helmchart
+    spec:
+      url: 'https://grafana.github.io/helm-charts'
+      name: 'grafana-agent-operator'
+
+targets:
+  chart:
+    name: Bump Chart Version
+    kind: yaml
```

```suggestion
    kind: yaml
    scmid: github
```

Probably missing scm configuration here too like in the next target